src/libponyrt/sched/start.c

```diff
@@ -98,12 +125,12 @@ static int parse_opts(int argc, char** argv, options_t* opt)
   {
     switch(id)
     {
-      case OPT_THREADS: opt->threads = atoi(s.arg_val); break;
-      case OPT_MINTHREADS: opt->min_threads = atoi(s.arg_val); minthreads_set = true; break;
+      case OPT_THREADS: if(parse_uint(&opt->threads, 1, s.arg_val)) err_out(id, "can't be less than 1"); break;
```

Comment:
Should this be `0` or `1` as the min value? `0` (or unspecified) used to mean physical core count… but did people really use `0` _explicitly_ to mean that?

Reply:
So we can scale down to 0 scheduler threads which means that minthreads is ok to be set to 0.

ponythreads with scaling on is really a maxthreads.

So, I think 1 as probably because, at 0, nothing can ever happen. we need to be able to do at least 1.